events/smart-descriptions.json

```
@@ -1,4 +1,39 @@
 {
+  "stormshield network security":
+  [
+    {
+      "value": "Connection accepted by rule {rule.id} from {source.ip} to {destination.ip}",
+      "conditions": [
+        {
+          "field": "stormshield.filter.action",
+          "value": "pass"
+        }
+      ],
+      "relationships": [
+        {
+          "source": "source.ip",
+          "target": "destination.ip",
+          "type": "connected was allowed"
```

To be consistent with other smart descriptions
```suggestion
          "type": "connected to"
```